Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement.lvproj

```diff
@@ -128,6 +156,7 @@
 				<Property Name="Source[8].sourceInclusion" Type="Str">TopLevel</Property>
 				<Property Name="Source[8].type" Type="Str">VI</Property>
 				<Property Name="SourceCount" Type="Int">9</Property>
+				<Property Name="TgtF_fastFileFormat" Type="Bool">true</Property>
```

Comment:
This is the only change I made to the project file. Everything else is LabVIEW's doing.